data/ecosystems/n/napster.toml

```diff
@@ -0,0 +1,6 @@
+# Ecosystem Level Information
+title = "napster"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/napster"]

```

Comment:
Any more support for Napster building directly on Algorand?  The open source projects don't mention Algorand at first glance.

Reply:
Hi Curtis,
Napster is owned by Algorand and Hivemind (since May 2022), and they have web3 upgrades planned for Q2 and Q3 on algorand rails. Ownership discussed here in wikipedia: https://en.wikipedia.org/wiki/Napster#:~:text=algorand , and this recent article: https://decrypt.co/121341/napster-acquires-mint-songs-to-advance-its-web3-ambitions .
It has been very under the radar for sure..

David

Reply:
ps. Not sure if i am supposed to mark this as resolved or leave for you to confirm..!

Thanks David